cmd/icingadb/main.go

```diff
@@ -64,14 +60,14 @@ func run() int {
 	}
 	defer db.Close()
 	{
-		logger.Infof("Connecting to database at '%s'", utils.JoinHostPort(cmd.Config.Database.Host, cmd.Config.Database.Port))
+		logger.Infof("Connecting to Database at '%s'", db.GetAddr())
```

Comment:
Not sure if this is upper-cased because it was forgotten from the previous iteration or for consistency with Redis being written in upper-case. I'd keep it lower-cased here as database is not a proper name.